src/test/groovy/org/gradle/android/CompilerArgsProcessorTest.groovy

```diff
@@ -0,0 +1,36 @@
+package org.gradle.android
+
+import org.gradle.api.Project
+import spock.lang.Specification
+
+import static org.gradle.android.CompilerArgsProcessor.Skip
+import static org.gradle.android.CompilerArgsProcessor.SkipNext
+
+class CompilerArgsProcessorTest extends Specification {
```

Comment:
Could you add some tests for `AnnotationProcessorOverride`, too?

Reply:
OK.